Orange/widgets/visualize/owsieve.py

```diff
@@ -167,13 +167,10 @@ def set_data(self, data):
             self.domain_model.set_domain(None)
         else:
             self.domain_model.set_domain(data.domain)
-            if any(attr.is_continuous for attr in data.domain):
```

Comment:
The idea here was to avoid unnecessary calls to `Discretize`. Wouldn't it be better to just replace the above condition with something like

    if any(attr.is_continuous for attr in chain(data.domain, data.domain.metas)):

and leave the rest as it is?